src/static/menu-mac.json

```diff
@@ -248,7 +248,30 @@
                             "shift": true
                         }
                     }
-                }
+                },
+                {
+                    "separator": true
+                },
+                {
+                    "id": "COPY_LAYER_STYLE",
+                    "shortcut": {
+                        "keyChar": "c",
```

Comment:
When rebasing, turn these into `keyChar: true` and then add the appropriate entries to `shortcuts-mac` and `-win`.